client/lib/site/utils.js

```diff
@@ -113,14 +123,43 @@ export default {
 			return false;
 		}
 
-		if ( site.options.is_multi_network ) {
+		const options = site.options;
+
+		if ( options.is_multi_network ) {
 			return false;
 		}
 
-		if ( site.options.file_mod_disabled ) {
+		if ( options.file_mod_disabled
+			&& ( options.file_mod_disabled.includes( 'disallow_file_mods' )
```

Comment:
we can't use 'includes' yet.. it's only working for chrome 47+ (and for example, I'm still on chrome 46), and it doesn't work at all for any explorer version :/